packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -27,6 +30,16 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
   const [selection, setSelection] = React.useState(new Selection({ onSelectionChanged: () => _onSelectionChanged() }));
   const [focusedItemIndices, setFocusedItemIndices] = React.useState(selection.getSelectedIndices() || []);
   const { suggestions, selectedSuggestionIndex, isSuggestionsVisible } = props.floatingSuggestionProps;
+  const [draggedItem, setDraggedItem] = React.useState<T>();
+  const dragDropHelper = new DragDropHelper({
+    selection: selection,
+  });
+
+  const theme = getTheme();
+  const dragEnterClass = mergeStyles({
+    backgroundColor: theme.palette.neutralLight,
+  });
```

Comment:
It is good to have a default but i think we should also give the option to the caller ( owa ) to override this if needed. I would make this dependent on some sort of props - basically something like using what comes as part of props from the caller or return this in _OnDragEnter.